<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCP STUB</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-17</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>981</width>
      <height>581</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>981</width>
      <height>581</height>
     </size>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Setting&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="lblInterfaceList_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>241</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>INTERFACE LIST</string>
      </property>
      <property name="textFormat">
       <enum>Qt::LogText</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>413</x>
        <y>110</y>
        <width>20</width>
        <height>431</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>751</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>31</width>
        <height>411</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QTextBrowser" name="txtBrowserInterface">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>121</width>
        <height>151</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>551</height>
       </rect>
      </property>
      <property name="title">
       <string>SETTING</string>
      </property>
      <widget class="QLabel" name="lblScenarioName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>SCENARIO NAME</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblCatagory">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>CATAGORY</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblProtocol">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>PROTOCOL</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblServer">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>214</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>SERVER</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lneEditScenarioName">
       <property name="geometry">
        <rect>
         <x>111</x>
         <y>37</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lneEditCatagory">
       <property name="geometry">
        <rect>
         <x>113</x>
         <y>94</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lneEditProtocol">
       <property name="geometry">
        <rect>
         <x>114</x>
         <y>160</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line_6">
       <property name="geometry">
        <rect>
         <x>103</x>
         <y>30</y>
         <width>20</width>
         <height>311</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lneEditServer">
       <property name="geometry">
        <rect>
         <x>113</x>
         <y>220</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lblPortNo">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>272</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>PORT NO:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lneEditPortNo">
       <property name="geometry">
        <rect>
         <x>112</x>
         <y>272</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>241</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>241</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>241</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>241</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_11">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>320</y>
         <width>241</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>24</y>
         <width>251</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pshButtonSaveScenariocredentials">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>352</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>SAVE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pshBtnGenDataFromTcpDump">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>430</y>
         <width>221</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>Gen From TCPDUMP</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="grpBoxStartSniffing">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>691</width>
        <height>531</height>
       </rect>
      </property>
      <property name="title">
       <string>SNIFFING</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>INTERFACE :</string>
       </property>
      </widget>
      <widget class="Line" name="line_13">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>701</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_14">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>701</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbBoxInterfaces">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>32</y>
         <width>201</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pshButtonStartSniffing">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>30</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>START SNIFFING</string>
       </property>
      </widget>
      <widget class="Line" name="line_15">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>20</y>
         <width>20</width>
         <height>61</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="line_16">
       <property name="geometry">
        <rect>
         <x>523</x>
         <y>30</y>
         <width>20</width>
         <height>51</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pshButtonStopSniffing">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>30</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>STOP SNIFFING</string>
       </property>
      </widget>
      <widget class="Line" name="line_17">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>20</width>
         <height>51</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="line_18">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>30</y>
         <width>20</width>
         <height>51</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="line_19">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>701</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QFrame" name="frmPacketSetting">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>691</width>
         <height>461</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QTableView" name="tblViewShowPacketSettings">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>681</width>
          <height>481</height>
         </rect>
        </property>
       </widget>
       <widget class="QScrollBar" name="horizontalScrollBar">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>450</y>
          <width>160</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QFrame" name="frmPacket">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>821</width>
        <height>501</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTableView" name="tblViewPacket">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>-10</y>
         <width>661</width>
         <height>471</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pshButtonReturnPacketSetting">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>460</y>
         <width>93</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>RETURN</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>460</y>
         <width>131</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>ADD NEW DATA</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QFrame" name="frmStartServer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="lblSelectScenario">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Scenario</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pshButtonStartServer">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>180</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>START SERVER</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblServerIp">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Server Ip</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblPOrt">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnEditPort">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>140</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnEditScenarioname">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>131</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnEditServerIp">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>131</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line_20">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>20</width>
        <height>551</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frmServerResult">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>90</y>
        <width>681</width>
        <height>441</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTableView" name="tblViewShowPacketSettingsNewPacket">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>31</y>
         <width>651</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtEditQuery">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>531</width>
         <height>341</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pshBttnCancelQuery">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>400</y>
         <width>93</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pshBttnUpdateQuery">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>400</y>
         <width>93</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>UPDATE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pshButtnInsertQuery">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>400</y>
         <width>93</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>INSERT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pshBttnDelete">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>400</y>
         <width>93</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>DELETE</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frmStopServer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>241</width>
        <height>271</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pshButtonSttopServer">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>121</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>STOP SERVER</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblIncomingPacketNumbers">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Incoming Packet</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblOutgoingPacketsNumber">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>80</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Outgoing Packet</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnEditIncomingPacketNumber">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>10</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnEditOutgoingPacketNumber">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>70</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pshButtonDissectPackets">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>2</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>DISSECT PACKET</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1077</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_Project"/>
    <addaction name="action_Open_Project"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="actionNewproject">
   <property name="text">
    <string>Newproject</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>OpenProject</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="action_New_Project">
   <property name="text">
    <string>&amp;New Project</string>
   </property>
  </action>
  <action name="action_Save_Project">
   <property name="text">
    <string>&amp;Save Project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Open_Project">
   <property name="text">
    <string>&amp;Open Project</string>
   </property>
  </action>
  <action name="actionAdd_Db_Server">
   <property name="text">
    <string>Add Db Server</string>
   </property>
  </action>
  <action name="actionAdd_App_Server">
   <property name="text">
    <string>Add App Server</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
